### ISSUE TITLE
Required gstreamer plugin 'libav' not found

### ISSUE THREAD

**Issue by tataudat** [antimof/UxPlay#34]
Got this error when trying to run the compiled binary. All the dependencies are already installed. Am I missing something here?

**Comment by doob09** [comment]
I am using Ubuntu 20.10 with AMD process and Nvidia 1060. I got the same problem when try to use the UxPlay .But You should try to use the original project : 
https://github.com/FD-/RPiPlay

Even It mentions it is for Raspberry pi . However, it works for me. Good luck !

**Comment by doob09** [comment]
> Got this error when trying to run the compiled binary. All the dependencies are already installed. Am I missing something here?

I am using Ubuntu 20.10 with AMD process and Nvidia 1060. I got the same problem when try to use the UxPlay .But You should try to use the original project :
https://github.com/FD-/RPiPlay

Even It mentions it is for Raspberry pi . However, it works for me. Good luck !

**Comment by fduncanh** [comment]
there is an updated  UxPlay pr #58. maybe you didnt install the libav plugin?

**Comment by yaylymov** [comment]
> Got this error when trying to run the compiled binary. All the dependencies are already installed. Am I missing something here?

This helped me: https://narkive.com/tlRb4aoQ.4
tl;dr:
Remove ~/.cache/gstreamer-1.0/registry.*.bin and try running ./uxplay again. It should work then.
